LICENSE.txt

```diff
@@ -200,3 +200,824 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
```

Comment:
@daisy-ycguo please add a list of references from where you composed following list of dependencies, for example, `Godeps.json` or `build.gradle` etc.